From d18af800f492323775fee0a3b8512920365fadcb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Thu, 8 Jan 2026 20:46:53 +0100
Subject: [PATCH 14/14] Modify the pid file name
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Use hailort8_service.pid as the pid file name in hailort_service.

This helps with installing it in parallel with the other hailort
version.

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [oe specific]
---
 hailort/hailort_service/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hailort/hailort_service/CMakeLists.txt b/hailort/hailort_service/CMakeLists.txt
index 609f4e7..035d92f 100644
--- a/hailort/hailort_service/CMakeLists.txt
+++ b/hailort/hailort_service/CMakeLists.txt
@@ -67,7 +67,7 @@ install(
 
 set(DAEMON_PID_DIR "/run/hailo")
 add_definitions(-DHAILO_DAEMON_PID_DIR="${DAEMON_PID_DIR}")
-add_definitions(-DHAILO_DAEMON_PID_FILE="${DAEMON_PID_DIR}/hailort_service.pid")
+add_definitions(-DHAILO_DAEMON_PID_FILE="${DAEMON_PID_DIR}/hailort8_service.pid")
 
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     # Create empty directory for hailort log file
-- 
2.52.0

